packages/backend-secret/API.md

```diff
@@ -11,6 +11,9 @@ import { BackendIdentifier } from '@aws-amplify/plugin-types';
 // @public
 export const getSecretClient: (secretClientOptions?: SecretClientOptions) => SecretClient;
 
+// @public
+export const getSecretClientWithAmplifyErrorHandling: (secretClientOptions?: SecretClientOptions) => SecretClient;
+

```

Comment:
Would `cli-core` be better place for this? 

Do we do any "amplify error handling" outside of "cli-ish" packages?

Reply:
On the other hand. We'd need to put dependency on backend-secret in cli-core which seems wrong. So let's keep it here.

One last question. Should we introduce `getSecretClientWithAmplifyErrorHandling` or add `enableAmplifyErrorHandling` to `secretClientOptions` ?